.github/workflows/build.yml

```diff
@@ -37,7 +34,7 @@ jobs:
           - os: macos-latest
             ghc: system
           - os: windows-latest
-            ghc: system
+            ghc: '9.2'
```

Comment:
@mpilgrem I think I remember that you opened a GHC issue for the windows build failure. What's the related issue number?